test/test-rotateElement.js

```diff
@@ -0,0 +1,203 @@
+var assert = require('chai').assert;
+var tetris = require('../tetris')
+describe("element rotation around the axis 3x3", function () {
+
+    it("element move the top", function () {
+        let elementTetromino = tetris.getCoordinates(`
+            ---
+            --X
+            ---
+        `)
+
+        let fulcrum = tetris.getCoordinates(`
+            ---
+            -X-
+            ---
+        `)
+        let stepElementTetromino = tetris.rotateElement(fulcrum, elementTetromino)
```

Comment:
Please make rotateElement inputs be objects of coordinates, instead ifarrays of such objects
You can take zeroes element of the result of getCoordinates

Reply:
Apply to all places

Reply:
I did not understand it:
"You can take zeroes element of the result of getCoordinates"